src/NUnitConsole/nunit3-console.tests/ConsoleRunnerTests.cs

```diff
@@ -11,10 +11,15 @@ namespace NUnit.ConsoleRunner.Tests
 {
     class ConsoleRunnerTests
     {
-        [Test]
+        [Test, Ignore("Needs to be rewritten")]
```

Comment:
I'm going to create an issue for this one - as a start. I'm not sure if anyone will come back to it, but it makes me more comfortable having it logged! 😄